packages/e2e/src/seeds/main.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { SeedServiceFactory } from "./factories"
+import { SeedDataResult, SeedParams } from "./types"
+
+const seedService = new SeedServiceFactory().create()
+
+let seedResult: SeedDataResult
+
+export const seedData = async (specFileName: string) => {
+  const data = getData(specFileName)
+
+  if (data) {
+    console.log(`Adding seed data to ${specFileName}`, data)
+    const result = await seedService.seed(data)
+    seedResult = result
+  }
+}
+
+// TODO [mw] implement it - scope of the next PR
+const getData = (specFileName: string): SeedParams | undefined => {
+  return
+}
+
+export const afterDataSeed = async () => {
```

Comment:
```suggestion
export const removeDataSeed = async () => {
```

The current function name, no descibe what the function do but why is used 😉

Reply:
Makes sense. Changed in https://github.com/mudita/mudita-center/pull/943/commits/e3ce4e2764516dcce32543af53f0d81508f1351a